price-estimator/src/models.rs

```diff
@@ -80,8 +80,13 @@ pub struct MarketsResult {
 }
 
 /// Sort orders by their price and merge orders with the same price into one.
-fn sanitize_orders(mut orders: Vec<TransitiveOrder>) -> Vec<TransitiveOrder> {
-    orders.sort_unstable_by(|a, b| a.price.partial_cmp(&b.price).unwrap_or(Ordering::Less));
+/// If reverse is false sorts by price ascending, if true descending.
+fn sanitize_orders(mut orders: Vec<TransitiveOrder>, reverse: bool) -> Vec<TransitiveOrder> {
```

Comment:
Nit: seeing that there is a bunch going on in the comments of this method. Maybe the flag could be called `descending` in which case we don't have to explain what we mean by reverse. Also `sanitize` seem to be a very general term meaning

> make clean and hygienic; disinfect.

How about naming this method `sort_and_aggregate_orders_by_price`?